app/views/users/index.html.erb

```diff
@@ -9,6 +9,7 @@
           <th>Email</th>
           <th></th>
           <th></th>
+          <th></th>
```

Comment:
Yeah, that should work :) `colspan` makes sure the `td` or `th` spans 3 columns.